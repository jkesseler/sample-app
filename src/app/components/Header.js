import React, { memo } from 'react';

import {
  AppBar, Toolbar, Typography,
} from '@material-ui/core';

export const Header = () => (
  <AppBar color="primary" position="static" style={{ height: 64 }}>
    <Toolbar style={{ height: 60 }}>
      <Typography color="inherit">Todo App</Typography>
    </Toolbar>
  </AppBar>
);

export default memo(Header);
