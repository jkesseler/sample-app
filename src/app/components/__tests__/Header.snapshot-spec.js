import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

// The named export is non-memoized component
import { Header } from '../Header';

describe('Header', () => {
  it('renders without crashing', () => {
    const component = shallow(<Header />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
