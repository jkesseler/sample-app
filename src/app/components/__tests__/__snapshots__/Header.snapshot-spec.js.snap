// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<WithStyles(AppBar)
  color="primary"
  position="static"
  style={
    Object {
      "height": 64,
    }
  }
>
  <WithStyles(Toolbar)
    style={
      Object {
        "height": 60,
      }
    }
  >
    <WithStyles(Typography)
      color="inherit"
    >
      Todo App
    </WithStyles(Typography)>
  </WithStyles(Toolbar)>
</WithStyles(AppBar)>
`;
